docs/DeveloperGuide.md

```diff
@@ -147,38 +195,45 @@ Classes used by multiple components are in the `seedu.addressbook.commons` packa
 ## **Implementation**
 
 This section describes some noteworthy details on how certain features are implemented.
-### [In Progress] \[DateTime\]
+### [Proposed] \[Date and Time verfication\]
 
 #### Proposed Implementation
-The implementation allows users to parse and compare unique DateTime types. 
+The implementation allows users to parse and compare unique Date and Time types.
 
+![Structure of DateTime and Time implementations](images/DateTimeClass.png)
+
+**Time type**:
+To parse, Time should be in the following format: **`HH:mm`** 
+* Single digits fields must include leading zero: `01:10`.
+
+Time will throw a parsing error if
+* `00:00:59` Format is wrong (e.g missing or additional digit).
+* `24:00` Invalid range (e.g invalid leap year).
+
+**DateTime type**:
 To parse, DateTime should be in the following format: **`dd-MM-yyyy HH:mm`** 
 * Single digits fields must include leading zero: `01-01-0101 01:10`.
-* Valid Calendar Range: \[`01-01-0001 00:00` - `31-12-9999 23:59`\].
+* Valid Calendar Range: \[`01-01-2019 00:00` - `31-12-9999 23:59`\].
 
 DateTime will throw a parsing error if
 * `1-10-2020 00:00:59` Format is wrong (e.g missing or additional digit).
 * `31-02-2020 00:00` Invalid range (e.g invalid leap year).
```

Comment:
I thinkt he example should come after the situation i.e. Format is wrong (e.g. 1-10 etc...)